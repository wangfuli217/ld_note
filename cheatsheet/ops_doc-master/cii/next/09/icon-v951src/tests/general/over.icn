procedure main()
   if not(&features == "large integers") then
      stop("large integers not supported")
   i := 100000 + 10000
   write(i)
   i +:= 2 ^ 30
   write(i)
   i +:= i
   write(i)
   i := 100000 * 10000
   write(i)
   i +:= 2 ^ 30
   write(i)
   i *:= i
   write(i)
   i := -100000 - 10000
   write(i)
   i +:= -(2 ^ 30)
   write(i)
   i -:= 2 ^ 30
   write(i)
end
