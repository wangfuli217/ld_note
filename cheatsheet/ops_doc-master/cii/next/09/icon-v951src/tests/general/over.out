110000
1073851824
2147703648
1000000000
2073741824
4300405152606846976
-110000
-1073851824
-2147593648
